integrationTests/testInitializer.go

```diff
@@ -1497,7 +1495,14 @@ func CreateNodesWithCustomStateCheckpointModulus(
 	idx := 0
 	for shardId := uint32(0); shardId < uint32(numOfShards); shardId++ {
 		for j := 0; j < nodesPerShard; j++ {
-			n := NewTestProcessorNodeWithStateCheckpointModulus(uint32(numOfShards), shardId, shardId, stateCheckpointModulus)
+			n := NewTestProcessorNode(ArgTestProcessorNode{
+				MaxShards:            uint32(numOfShards),
+				NodeShardId:          shardId,
+				TxSignPrivKeyShardId: shardId,
+				StateCheckpointModulus: &StateCheckpointModulus{
```

Comment:
interesting approach to be able to check "unset" primitive values.
Maybe we should have renamed the struct `StateCheckpointModulus` to `IntWrapper` so we could have used it elsewhere? We can have a .go file for all used wrappers like this one. 
Also, please check the indentation (L1501 vs L1502)

Reply:
added `IntWrapper` as suggested